src/components/pages/navigationBar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavigationLink, Navigation, NavigationItem } from './navigationBarItems';
+
+const NavigationBar = () => (
+  <>
+    <Navigation>
+      <NavigationItem>
+        <NavigationLink to="src/components/pages/Home.js" activeStyle>
+          Home
+        </NavigationLink>
+        <NavigationLink to="/components/Calculator" activeStyle>
+          Calculator
+        </NavigationLink>
+        <NavigationLink to="src/components/pages/quote.js" activeStyle>
+          Quote
+        </NavigationLink>
+      </NavigationItem>
+    </Navigation>
+  </>
+);
```

Comment:
* Kindly work on your navigation links as it was broken while working on the previous CR requested changes. below is a video of the error  

https://user-images.githubusercontent.com/52497246/200692608-44044d5c-bf5a-4fb9-bdfc-b2a6c7ff7362.mp4

Reply:
I changed the App.js so that it gets the same link as given in the navigation link. Now it is working properly.
Since that suggestion was "optional" I retained my original coding to get wireframes as requested.
Now the problems shown in the video are resolved. on clicking the navigation link all the pages are coming.